corporate/models.py

```diff
@@ -446,12 +446,14 @@ def __str__(self) -> str:
     def name_from_tier(tier: int) -> str:
         # NOTE: Check `statement_descriptor` values after updating this.
         # Stripe has a 22 character limit on the statement descriptor length.
+        # The exceptions are complimentary access plans, which are never
+        # billed through Stripe.
```

Comment:
It might be better to put this part of the comment explanation just above the relevant rows themselves, but I'm not certain one way or the other.

Reply:
Done.